pinniped-components/tanzu-auth-controller-manager/hack/generate-package-secret.sh

```diff
@@ -6,7 +6,6 @@
 set -euo pipefail
 
 MY_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
-TF_ROOT="${MY_DIR}/../../.."
```

Comment:
So there is one more file in this directory, the `run.sh` file.  Several things here:

It still contains a `TF_ROOT` var:
```
TF_ROOT="${MY_DIR}/../../.."
```
It references `<TF_ROOT>/packages/tanzu-auth`.  This is where the CARVEL package for the `tanzu-auth-controller-manager` lives.  

I think we need a conversation about this.

Reply:
@yharish991 I believe you had stated `<TF_ROOT>/packages` is going to stay put for now, but when Pinniped components move this work will have to be sheduled?